src/components/chat/comment-card.tsx

```diff
@@ -0,0 +1,100 @@
+import { inject, observer } from "mobx-react";
+import React from "react";
+import { BaseComponent } from "../base";
+import DocumentCommentIcon from "../../assets/document-id.svg";
+import SendIcon from "../../assets/send-icon.svg";
+import "./comment-card.scss";
+import "../themes.scss";
+
+interface IProps {
+}
+
+interface IState {
+  commentTextAreaHeight: number | string;
+  commentAdded: boolean;
+  commentText: string;
+}
+
+@inject("stores")
+@observer
+export class CommentCard extends BaseComponent<IProps, IState> {
+
+  constructor(props: IProps) {
+    super(props);
+    this.handleCommentTextAreaChange = this.handleCommentTextAreaChange.bind(this);
+    this.handleCancelPost = this.handleCancelPost.bind(this);
+    this.handleSendPost = this.handleSendPost.bind(this);
+
+    this.state = {
+      commentTextAreaHeight: 35,
+      commentAdded: false,
+      commentText: "",
+    };
+  }
+
+  public render() {
+    const { ui } = this.stores;
+    return (
+      <div className={`comment-card ${ui.activeNavTab}`} data-testid="comment-card">
+        {this.renderThreadHeader()}
+        {this.renderCommentTextbox()}
+      </div>
+    );
+  }
+
+  private renderThreadHeader() {
+    const { ui } = this.stores;
+    return (
+      <div className={`comment-card-header ${ui.activeNavTab}`}>
+        <DocumentCommentIcon className="new-thread-header-icon"/>
+      </div>
+    );
+  }
+  handleCommentTextAreaChange = (event: React.FormEvent<HTMLTextAreaElement>) => {
+    const target = event.currentTarget as HTMLTextAreaElement;
+    const targetText = target.value;
```

Comment:
Possibly related to the `FormEvent`/`ChangeEvent` issue above, I would have expected `event.target.value` to be the value of the `<textarea>` without any casting required. `currentTarget` is generally only needed when you need to distinguish between the element that generated the event (the `<textarea>` in this case) from the element on which the handler was placed, e.g. if you've attached the handler on a parent element. In this case they should be the same, in which case using `target` would be more conventional.